v1/pkg/client/api/openapi.yaml

```diff
@@ -4771,11 +4774,16 @@ components:
           CHAPSecret: CHAPSecret
           CHAPUserName: CHAPUserName
           InitiatorName: iqn.1234.com.hpe-initiator
+        FC:
+          HostIPAddress: 10.1.2.3
+          InitiatorName: 51:40:2e:c0:17:32:9a:e8
```

Comment:
same here.

Reply:
If this comment is for HostIPAddress is not needed - please check https://github.com/hpe-hcss/quake/blob/1638e3643ed0be37a15e3148003bd8036fa1ef44/portal/pkg/volumeattachment/translate.go#L54.